VERSIONING.md

```diff
@@ -0,0 +1,19 @@
+In order to support backwards and forwards compatibility, SBP messages
+are versioned and follow a small number of conventions, enabling
+devices to indicate understood message formats, capabilities and
+features. SBP uses a major, minor numbering scheme to indicate
+versions, where major and minor are represented by 8-bit values. The
+minor number is incremented by each message addition, deprecation. The
+major number is incremented when the format of a message is
+changed.
+
+Message ID's are guaranteed to be unique for all minor versions in a
```

Comment:
Agree, I think the content is spot on but could be restructured to be easier to read.